src/development/LocalTest/LocalTest.csproj

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
+    <UserSecretsId>56f36ce2-b44b-415e-a8a5-f399a76e35b9</UserSecretsId>
```

Comment:
What effect does this have?

Reply:
It is simply a unique id so that the user-secrets feature knows which secrets file to look for when loading settings. It is exclusive for Development.